ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Cross-Chain Tracking Standard
+description: This EIP standardizes a simplified set of cross-chain bridge transaction event formats, aiming to provide fundamental traceability standard for cross-chain transactions.
+author: Shulei（@baishuo13）<shulei.shu@ant-intl.com>, Jeff Fei (@77eff) <j.fei@ant-intl.com>, Kenny Kung (@kennyk10) <kenny.kung@ant-intl.com>
+discussions-to: 
```

Comment:
Please create a discussions topic in Eth Magicians linking to this PR
https://ethereum-magicians.org/c/ercs/57